go.opentelemetry.io/vanity.yaml

```diff
@@ -1,7 +1,11 @@
 host: go.opentelemetry.io
 
 paths:
-  /otel:
+  /lib:
```

Comment:
-1 on changing this

Reply:
Why?

Reply:
But if you prefer "/otel" and everyone has that opinion I am fine to keep that. It just does not make too much sense because it just duplicates the same information from the URL, not specifying that it is from the library.